cron/format/json.go

```diff
@@ -69,10 +69,21 @@ type jsonScorecardV2 struct {
 	Commit  string `json:"commit"`
 }
 
+type jsonFloatScore float64
+
+func (s jsonFloatScore) MarshalJSON() ([]byte, error) {
```

Comment:
Might skip this, just X.0 is fine for even ints.

Reply:
+1. Given how floats are represented I'm unsure this will work reliability in every case either.

Reply:
the BQ has a type `float`, and JSON sets this field with type `number`. (Note: score are 1-10)

Reply:
why not keep everything float ?

Reply:
JSON does not understand the distinction between `float` and `int` in general. The schema we export (if users use it) has the field typed as `number` and not an integer.. so it's not an integer. Removing the `.X` was mostly esthetic when a human being looks at it. I'm fine to keep all `X.Y`... it may even be better: I can imagine users make it an `int` by mistake and only later realize it should be a `float`.

Reply:
Yes that is what i meant, keep everything float.